vlingo-reactive-messaging-patterns/src/main/java/io/vlingo/reactive/messaging/patterns/messagebus/CommandHandler.java

```diff
@@ -0,0 +1,97 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+package io.vlingo.reactive.messaging.patterns.messagebus;
+
+/**
+ * CommandHandler
+ *
+ * @author brsg.io
+ * @since Oct 31, 2018
+ */
+public class CommandHandler
+{
+    private final String commandId;
+    private final String applicationId;
+    private final TradingProcessor tradingProcessor;
+    
+    public CommandHandler( String commandId, String applicationId, TradingProcessor tradingProcessor )
```

Comment:
It seems to me that `CommandHandler` would serve much better if it was able to be told to do something with some parameters rather than being looked up and queried for information. Handling something is what its name says that it does, so why not make it do that?